recipes/gmpy2/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "2.0.7" %}
+
+package:
+  name: gmpy2
+  version: {{ version }}
+
+source:
+  fn: gmpy2-{{version}}.zip
+  url: https://pypi.io/packages/source/g/gmpy2/gmpy2-{{ version }}.zip
+  md5: b5aada3ee5afb316ea94604f45192054
+
+build:
+  number: 0
+  skip: true        # [win and not py35]
+  features:
+    - vc14          # [win and py35]
+  script:
+    - python setup.py install --shared=$PREFIX                                          # [unix]
+    - python setup.py build_ext -f --shared=%LIBRARY_PREFIX% -DMSC_USE_DLL=1 install    # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gmp           # [unix]
+    - mpir          # [win]
+    - mpfr
+    - mpc
+    - python
+  run:
+    - gmp           # [unix]
+    - mpir          # [win]
+    - mpfr
+    - mpc
+    - python
+
+test:
+  requires:
+    - python 2.7.*  # [py27]
+    - python 3.4.*  # [py34]
+    - python 3.5.*  # [py35]
+  imports:
+    - gmpy2
+
+about:
+  home: https://pypi.python.org/pypi/gmpy2
+  license: LGPL 3
+  summary: 'GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x'
+
+extra:
+  recipe-maintainers:
+    - isuruf
```

Comment:
This can be done here or in the feedstock. Feel free to add @asmeurer as he has explicitly stated he would be interested in helping maintain `sympy` dependencies and this is one IIRC.

I'm ok with being here. Though I may not be as active just FYI.